search/quality_eval.go

```diff
@@ -698,7 +698,7 @@ func EvaluateQuery(q EvalQuery, serverUrl string, skipExpectations bool) EvalRes
 					}
 					if HitMatchesExpectation(hit, hitSource, q.Expectations[i]) {
 						rank = j + 1
-						if j <= 2 {
+						if j <= 2 || i > 2 {
```

Comment:
This condition is not enough. Ranks 6-9 also became 'good'.

Reply:
fixrd